state_machine/applications/flight/lib/mekf.py

```diff
@@ -0,0 +1,25 @@
+# Multiplicative Extended Kalman Filter
+# Based on Zac Manchester's Formulation
+# Writen by Aleksei Seletskiy
+try:
+    from ulab.numpy import block, matmul
+    import ulab.linalg as linalg
+except Exception:
+    from numpy import linalg, block, dot as matmul
+from lib.mathutils import L
+from math import cos, sin
+
+q = []  # Quaternion attitude vector
+β = []  # Gyro bias vector
+P = [[]]  # Covariance matrix
+# mutable struct EKF
+#     q::Vector{Float64}
+#     β::Vector{Float64}
+#     P::Matrix{Float64}
+# end
+
+def f(q, β, ω, δt):
```

Comment:
Same thing here; would prefer a more descriptive name for users who are not as well versed with the physics. At the same time, I have a feeling you may have chosen these function names to follow academic conventions, so this is up to your discretion.

Reply:
Good idea.

Reply:
Changed to `propogate_state`